test/test_nn.py

```diff
@@ -4022,7 +4022,7 @@ def test_spectral_norm(self):
         self.assertFalse('weight' in m._buffers)
         self.assertFalse('weight' in m._parameters)
         # it should also be sharing storage as `weight_orig`
-        self.assertEqual(m.weight_orig.storage(), m.weight.storage())
+        self.assertEqual(m.weight_orig.storage()._cdata, m.weight.storage()._cdata)
```

Comment:
From the comment I get that we actually want to make sure that both tensors share the same memory. The same C++ storage can have multiple Python identities so we need to make sure that they point to the same memory location.